ui/src/router/details/logs/RouterLogsView.js

```diff
@@ -55,24 +67,73 @@ export const RouterLogsView = ({ router }) => {
     (entries) =>
       !!entries.length ? entries[entries.length - 1].timestamp : undefined,
     (entries) => entries.map((entry) => LogEntry.fromJson(entry).toString()),
-    configOptions
+      appConfig.podLogs,
   );
 
   const availableComponents = components.filter(
     (c) => c.value === "router" || !!get(router, `config.${c.value}`)
   );
 
+  useEffect(
+    () => {
+      let urls = {}
+      if (appConfig.imagebuilder.cluster && appConfig.imagebuilder.gcp_project && appConfig.imagebuilder.namespace &&
+        currentProject) {
+        // set router url
+        urls["router"] = createStackdriverUrl({
+          gcp_project: environment.gcp_project,
+          cluster: environment.cluster,
+          namespace: currentProject.name,
+          pod_name: router.name + "-turing-router-"  + router.config.version,
+          start_time: router.updated_at,
+        }, "router", appConfig.imagebuilder)
+
+        // set enricher url
+        if (router.config.enricher.type === "docker") {
+          urls["enricher"] = createStackdriverUrl({
+            gcp_project: environment.gcp_project,
+            cluster: environment.cluster,
+            namespace: currentProject.name,
+            pod_name: router.name + "-turing-enricher-" + router.config.version,
+            start_time: router.updated_at,
+          }, "enricher", appConfig.imagebuilder)
+        }
+
+        // set ensembler url
+        if (router.config.ensembler.type === "docker" || router.config.ensembler.type === "pyfunc") {
+          urls["ensembler"] = createStackdriverUrl({
+            gcp_project: environment.gcp_project,
+            cluster: environment.cluster,
+            namespace: currentProject.name,
+            pod_name: router.name + "-turing-ensembler-" + router.config.version,
+            start_time: router.updated_at,
+          }, "ensembler", appConfig.imagebuilder)
+        }
+
+        // set image builder url
+        if (router.config.ensembler.type === "pyfunc" && ensembler) {
+          urls["ensembler_image_builder"] = createStackdriverUrl({
+            job_name: "service-" + currentProject.name + "-" + ensembler.name,
+            start_time: ensembler.updated_at,
```

Comment:
I set the `start_time` as the time after the `ensembler` has been updated because the `router.updated_at` time is always AFTER the image building jobs has been completed so there's no convenient timestamp I can use to set as the start time for the image building job.